testdata/xml-data/test-data-systematic.xml

```diff
@@ -200,9 +199,6 @@
             <text permissions="prop-default" encoding="xml">
                 Another text without salsah-links
             </text>
-            <text permissions="prop-default" encoding="xml">
-                Another text without salsah-links
-            </text>
```

Comment:
Duplicate values for the same property are not allowed. Apparently, there were some whitespaces that had coverted this up until now.